app/src/main/java/com/babylon/wallet/android/utils/AppEventBus.kt

```diff
@@ -13,7 +13,7 @@ import javax.inject.Singleton
 @Singleton
 class AppEventBus @Inject constructor() {
 
-    private val _events = MutableSharedFlow<AppEvent>(replay = 1)
+    private val _events = MutableSharedFlow<AppEvent>()
```

Comment:
Just make sure it won't affect other usages of this flow, as it won't emit the last value upon collection anymore.

Reply:
this was added when we worked on NPS survey, since initial set of requirements was to replay certain event when app start. This is no longer needed and might actually cause some trouble